modules/cluster-logging-loki-deploy.adoc

```diff
@@ -0,0 +1,128 @@
+
+.Prerequisites
+
+* {logging-title-uc} Operator 5.5 and later
+* AWS S3 bucket for log storage
+
+.Procedure
+
+. Install the `LokiOperator` Operator:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Choose  *LokiOperator* from the list of available Operators, and click *Install*.
+
+.. Under *Installation Mode*, select *All namespaces on the cluster*.
+
+.. Under *Installed Namespace*, select *openshift-operators-redhat*.
++
+You must specify the `openshift-operators-redhat` namespace. The `openshift-operators`
+namespace might contain Community Operators, which are untrusted and might publish
+a metric with the same name as an {product-title} metric, which would cause
+conflicts.
+
+.. Select *Enable operator recommended cluster monitoring on this namespace*.
++
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object.
+You must select this option to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
+
+.. Select an *Approval Strategy*.
++
+* The *Automatic* strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
++
+* The *Manual* strategy requires a user with appropriate credentials to approve the Operator update.
+
+.. Click *Install*.
+
+.. Verify that you installed the LokiOperator. Visit the *Operators* → *Installed Operators* page and look for *LokiOperator*.
+
+.. Ensure that *LokiOperator* is listed with *Status* as *Succeeded* in all the projects.
++
+. Create a `Secret` YAML file that uses the `access_key_id` and `access_key_secret` fields to specify your base64-encoded AWS credentials. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: logging-loki-s3
+  namespace: openshift-logging
+data:
+  access_key_id: QUtJQUlPU0ZPRE5ON0VYQU1QTEUK
+  access_key_secret: d0phbHJYVXRuRkVNSS9LN01ERU5HL2JQeFJmaUNZRVhBTVBMRUtFWQo=
+  bucketnames: czMtYnVja2V0LW5hbWU= <1>
+  endpoint: aHR0cHM6Ly9zMy5ldS1jZW50cmFsLTEuYW1hem9uYXdzLmNvbQ== <2>
+----
+<1> Base64-encoded bucket name
+<2> Base64-encoded S3 API endpoint
++
+. Create `LokiStack` custom resource:
++
+[source,yaml]
+----
+  apiVersion: loki.grafana.com/v1
+  kind: LokiStack
+  metadata:
+    name: logging-loki
+    namespace: openshift-logging
+  spec:
+    size: 1x.small
+    storage:
+      schemas:
+      - version: v12
+        effectiveDate: 2022-06-01
+      secret:
+        name: logging-loki-s3
+        type: s3
+    storageClassName: gp2
+    tenants:
+      mode: openshift-logging
+----
++
+.. Apply the configuration:
++
+[source,terminal]
+----
+oc apply -f logging-loki.yaml
+----
++
+. Create or edit a  Create or edit a `ClusterLogging` CR:
++
+[source,yaml]
+----
+  apiVersion: logging.openshift.io/v1
+  kind: ClusterLogging
+  metadata:
+    name: cr-lokistack
+    namespace: openshift-logging
+  spec:
+     logStore:
+       lokistack:
+         name: <lokistack-name>
```

Comment:
Can we mention the 'logging-loki' as the lokistack name since we have been using that a reference in earlier CR's.

Reply:
Corrected.